src/stack_aggregator.py

```diff
@@ -506,8 +519,22 @@ def get_dependency_data(epv_set):
         if result:
             epv_list['result']['data'] += result['result']['data']
 
+    # # Identification of unknown dependencies
+    epv_data = epv_list['result']['data']
+    logger.error("get_dependency_data")
+    logger.error(dep_list)
+    logger.error(epv_data)
+    for k, v in dep_list.items():
+        known_flag = False
+        for knowndep in epv_data:
+            if k == knowndep['package']['name'][0] and v == knowndep['version']['version'][0]:
+                known_flag = True
+                break
+        if not known_flag:
+            unknown_deps_list.append({'name': k, 'version': v})
+
     result = add_transitive_details(epv_list, epv_set)
-    return {'result': result}
+    return {'result': result, 'unknown_deps': unknown_deps_list}
```

Comment:
have not it needs to be reflected in unit tests?

Reply:
added

Reply:
@animuk would it be possible to also add a test that would check situation where there are unknown deps?